subcommand/server-acl-init/rules.go

```diff
@@ -233,6 +232,7 @@ node_prefix "" {
 {{- if .EnableNamespaces }}
 namespace_prefix "" {
 {{- end }}
+  acl = "write"
```

Comment:
The key is here. Moving this inside the `namespace_prefix ""` block.

Reply:
yes exactly. If it was `namespace "blah"` then it would be only for `blah` and if it was `namespace_prefix "blah-"` it would be for `blah-.*`